content/en/serverless/guide/azure_app_service_linux_sidecar.md

```diff
@@ -0,0 +1,169 @@
+ENTRYPOINT ["dotnet", "<your dotnet app>.dll"]
+```
+{{< /programming-lang >}}
+{{< /programming-lang-wrapper >}}
+
+Then, build the image and push it to your preferred container registry.
+
+### Create your Linux Web App
+
+1. In the Azure Portal, go to **App Services** and select **Create**.
+1. On the **Basics** tab, provide the required details.
+   - For **Publish**, select **Container**.
+   - For **Operating System**, select **Linux**.
+   Then, select **Next: Container >**.
+1. On the **Container** tab, provide the required details.
+   - For **Sidecar support**, select **Enabled**.
+   - For **Image Source**, select your chosen registry.
+   - Then, specify the **Registry**, **Image**, **Tag**, and **Port** for your container image.
+1. Select **Review + create**, then select **Create**.
+
+### Add Datadog environment variables
+
+In the Azure portal, select your app. In the left menu, select **Configuration** > **Application settings**. Then, add the following environment variables as application settings.
+
+
+`DD_API_KEY` 
+: Your [Datadog API key][3]. <br/>
+**Required**. Alternatively, you can source your API key (and other sensitive information) from Azure Key Vault. See [Use Key Vault references as app settings in Azure App Service][4].
+
+`DD_SITE` 
+: {{< region-param key="dd_site" code="true" >}} <br/>
+Corresponds to your [Datadog site][5]. Defaults to `datadoghq.com`.
+
+`DD_SERVICE` 
+: Supply a service name to be displayed in your Datadog [Service Catalog][6]. See [Unified Service Tagging][7]. 
+
+`DD_ENV` 
+: A name for your environment, such as `staging` or `prod`. See [Unified Service Tagging][7].
+
+`DD_SERVERLESS_LOG_PATH` 
+: `/home/Logfile/*.log` <br/>
+Corresponds to the path where you write your application logs. If you have changed this location, specify your custom location in this setting.
+
+`DD_DOTNET_TRACER_HOME` (**Required**)
+: `/datadog/tracer`
+
+`DD_TRACE_LOG_DIRECTORY` (**Required**)
+: `/home/Logfiles/dotnet`
+
+`CORECLR_ENABLE_PROFILING` (**Required**)
+: `1`
+
+`CORECLR_PROFILER` (**Required**)
+: `{846F5F1C-F9AE-4B07-969E-05C26BC060D8}`
+
+`CORECLR_PROFILER_PATH` (**Required**)
+: `/datadog/tracer/Datadog.Trace.ClrProfiler.Native.so`
+
+
+### Add the Datadog sidecar
+
+1. In the Azure portal, select your app. In the left menu, select **Deployment Center**.
+1. Select **Add**. Under **Add container**, provide the following details:
+   - **Name**: `datadog`
+   - **Image source**: Docker Hub or other registries
+   - **Image type**: Public
+   - **Registry server URL**: sitecontainerssampleacr.azurecr.io
+   - **Image and tag**: datadog-dotnet:2.0
+   - **Port**: 8126
+   Then, select **Apply**.
+
+## Next steps
+
+After you finish instrumenting your application, go to the [Serverless > Azure][8] page in Datadog to see your observability data. Your application logs are available in the [Log Explorer][9], and your traces are available in the [Trace Explorer][10].
```

Comment:
My brain doesn't want Log Explorer or Trace Explorer to have an article before them, but IDK if it's standard to do it this way.